colcon_core/shell/bat.py

```diff
@@ -26,6 +26,8 @@ class BatShell(ShellExtensionPoint):
     FORMAT_STR_USE_ENV_VAR = '%{name}%'
     FORMAT_STR_INVOKE_SCRIPT = 'call:_colcon_prefix_bat_call_script ' \
         '"{script_path}"'
+    FORMAT_STR_REMOVE_TRAILING_SEPARATOR = 'if "%{name}:~-1%==";" ' \
```

Comment:
The extension point should declare and document this constant.

Reply:
See de08e25 and d57b3b0